### PR TITLE
Titles are for losers. Guess I'm a loser then.

### PR DESCRIPTION
**Pull request by puppy0cam**
general improvements, and remove a redundant page that is a duplicate of my superior page for the list of queues.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -113,8 +113,15 @@ Access to pouch transactions: removed
 ```
 
 ---
- 
+Q: **How do I access a queue?**
 
+A: To access a queue, you must first ensure that you have had one granted. To be granted a queue, simply request a modification to your application with the additional queue added to it. After a queue has been granted to you, you can start consuming from it with `%username%_%queue%`.
+
+*Note: Failiure to actually consume from your queue can result in it being revoked*
```
Comment (louietyj) [inline index.md]: Nit typo: "Failure"

**Review by louietyj — CHANGES_REQUESTED** [review]

File: results.md [hunk 2]
```diff
@@ -8,11 +8,11 @@ navigation: 5
 Possible results are:
 - `Ok` - Everything is Ok
 - `BadAmount` - Amount is either less than or equal to zero
-- `BadCurrency` - The currency you chose is not allowed
+- `BadCurrency` - The currency you specified is not permitted to be used in this request.
 - `BadFormat` - Message format is bad. It could be an invalid javascript, or types are wrong, or not all fields are sane
 - `ActionNotFound` - the action you have requested is absent. Check spelling
 - `NoSuchUser` - UserID is wrong, or user became inactive
-- `NotRegistered` - Your app is not yet registered
+- `NotRegistered` - Your app is not yet registered. You must request access to this feature seperately.
```
Comment (louietyj) [inline results.md]: Nit typo: "separately"

**Review by louietyj — CHANGES_REQUESTED** [review]

**Comment by puppy0cam** [comment]
This pull request now includes the LevelIsLow result

**Review by sad-unicorn — APPROVED** [review]
plz rebase

**Comment by sad-unicorn** [comment]
I @puppy0cam, I'm not can into rebases!

**Pull request merged**